config/core/resources/eventtype.yaml

```diff
@@ -71,7 +71,7 @@ spec:
       JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
   versions:
   - name: v1alpha1
```

Comment:
delete? the webhooks are disabled and it is not like we can flip this back on.

Reply:
you can't remove them or crd validation will complain when removing that version. I need to see if there's something else missing.